lib/ruby_lsp/event_emitter.rb

```diff
@@ -2,6 +2,18 @@
 # frozen_string_literal: true
 
 module RubyLsp
+  # EventEmitter is an intermediary between our requests and Syntax Tree visitors. It's used to visit the document's AST
+  # and emit events that the requests can listen to for providing functionality. Usages:
```

Comment:
Let's separate the usages list with a new line before and after, it will make the comment easier to read.

Reply:
Done!